packages/squiggle-lang/src/rescript/Utility/Operation.res

```diff
@@ -62,6 +63,7 @@ module Error = {
     switch err {
     | DivisionByZeroError => "Cannot divide by zero"
     | ComplexNumberError => "Operation returned complex result"
+    | InfinityError => "Operation returned + or - infinity"
```

Comment:
"Operation returned infinity or negative infinity". For that matter, we could just make two errors; one for each.